src/main/java/net/dries007/tfc/common/entities/ai/amphibian/AmphibianAi.java

```diff
@@ -76,28 +84,55 @@ private static void initCoreActivity(Brain<? extends AmphibiousAnimal> brain)
                 ImmutableSet.of(),
                 GateBehavior.OrderPolicy.ORDERED,
                 GateBehavior.RunningPolicy.TRY_ALL,
                 ImmutableList.of(
-                        Pair.of(StartAttacking.create(AmphibianAi::getAttackTarget), 2),
-                        Pair.of(new Swim(0.5F), 5),
-                        Pair.of(RandomStroll.stroll(0.15F, false), 2),
-                        Pair.of(SetWalkTargetFromLookTarget.create(AmphibianAi::canSetWalkTargetFromLookTarget, AmphibianAi::getSpeedModifier, 3), 3),
-                        Pair.of(new DoNothing(30, 60), 3),
-                        Pair.of(StrollToPoi.create(MemoryModuleType.HOME, 0.15F, 5, 100), 3)
+                    Pair.of(StartAttacking.create(AmphibianAi::getAttackTarget), 2),
+                    Pair.of(RandomStroll.swim(0.5F), 2),
+                    Pair.of(RandomStroll.stroll(0.15F, false), 2),
+                    Pair.of(SetWalkTargetFromLookTarget.create(AmphibianAi::canSetWalkTargetFromLookTarget, AmphibianAi::getSpeedModifier, 3), 3),
+                    Pair.of(new DoNothing(30, 60), 3)
+                    //This behavior seems to be broken, disabling it for now
+                    //Pair.of(StrollToPoi.create(MemoryModuleType.HOME, 0.15F, 5, 100), 3)
                 )
             ))
-        );
+        ));
     }
 
+//    Vanilla Axolotl AI for reference
+//    private static void initIdleActivity(Brain<Axolotl> p_149309_) {
+//        p_149309_.addActivity(Activity.IDLE, ImmutableList.of(
+//            Pair.of(0, SetEntityLookTargetSometimes.create(EntityType.PLAYER, 6.0F, UniformInt.of(30, 60))),
+//            Pair.of(1, new AnimalMakeLove(EntityType.AXOLOTL, 0.2F)),
+//            Pair.of(2, new RunOne<>(ImmutableList.of(
+//                Pair.of(new FollowTemptation(AxolotlAi::getSpeedModifier), 1),
+//                Pair.of(BabyFollowAdult.create(ADULT_FOLLOW_RANGE, AxolotlAi::getSpeedModifierFollowingAdult), 1))
+//            )),
+//            Pair.of(3, StartAttacking.create(AxolotlAi::findNearestValidAttackTarget)),
+//            Pair.of(3, TryFindWater.create(6, 0.15F)),
+//            Pair.of(4, new GateBehavior<>(
+//                ImmutableMap.of(MemoryModuleType.WALK_TARGET, MemoryStatus.VALUE_ABSENT),
+//                ImmutableSet.of(),
+//                GateBehavior.OrderPolicy.ORDERED,
+//                GateBehavior.RunningPolicy.TRY_ALL,
+//                ImmutableList.of(
+//                    Pair.of(RandomStroll.swim(0.5F), 2),
+//                    Pair.of(RandomStroll.stroll(0.15F, false), 2),
+//                    Pair.of(SetWalkTargetFromLookTarget.create(AxolotlAi::canSetWalkTargetFromLookTarget, AxolotlAi::getSpeedModifier, 3), 3),
+//                    Pair.of(BehaviorBuilder.triggerIf(Entity::isInWaterOrBubble), 5),
+//                    Pair.of(BehaviorBuilder.triggerIf(Entity::onGround), 5))))));
+//    }
```

Comment:
Rather than doing this, add a javadoc with an `@link` tag to the vanilla code here. Having a large section of copy-pasted vanilla code is just asking for it to get out of date, and it's also non navigable (you can't navigate into any of these symbols)

Reply:
Oops, meant to delete this before merging, but I will @link it, good idea